ckanext/datastore/backend/postgres.py

```diff
@@ -572,6 +570,9 @@ def _build_query_and_rank_statements(
     return statement, rank_statement
 
 
+def _get_timeout() -> int:
+    return config.get('ckan.datastore.default_query_timeout', 60000)
```

Comment:
The default is set in config_declaration so we don't need to repeat it here.

Reply:
Done here https://github.com/avdata99/ckan/commit/3d99079f7a8c5cab4efcb98a05d96ee6bf95f1b4